doc/Language/objects.pod6

```diff
@@ -263,13 +260,13 @@ Method names can be resolved at runtime with the C<.""> operator.
 =head2 self
 
 Inside a method, the term C<self> is available, which is bound to the
-invocant, i.e.  the object that the method was called on. C<self> can be
+invocant, the object that the method was called on. C<self> can be
```

Comment:
This change removes “i.e.” but doesn't replace it with anything with the same meaning (in other words). Should leave it as before or change the rest of the sentence to be written without it.